pjsip/src/pjsua-lib/pjsua_vid.c

```diff
@@ -1064,9 +1079,7 @@ pj_status_t pjsua_vid_channel_update(pjsua_call_media *call_med,
 
 #if defined(PJMEDIA_STREAM_ENABLE_KA) && PJMEDIA_STREAM_ENABLE_KA!=0
 	/* Enable/disable stream keep-alive and NAT hole punch. */
-	si->use_ka = acc->cfg.use_stream_ka;
-
-        si->ka_cfg = acc->cfg.stream_ka_cfg;
+	si->use_ka = pjsua_var.acc[call->acc_id].cfg.use_stream_ka;
```

Comment:
looks a bit strange, is this something related to the codec impl? also the changes below.